src/java_bytecode/generate_java_generic_type.cpp

```diff
@@ -11,6 +11,15 @@
 #include <java_bytecode/java_types.h>
 #include <java_bytecode/java_utils.h>
 
+static std::string pretty_print_java_type(const std::string &fqn_java_type)
```

Comment:
Could you add some Doxygen to this, in particular, with some example of the input/output of the function? I'm also wondering if this is really the best name for this function, because it seems to me its doing more than just pretty printing if I'm reading this code correctly? It's striping off the leading `java::java.lang`  - from java types that include that, so maybe the function name should make that clearer?

Reply:
A suggestion for naming: `strip_package_from_java_class` ??? That's just a random suggestion, but it might be nice if the name was somewhat orthogonal to the exisiting `java_class_to_package` function that gets called below.